plugins/inputs/tail/multiline_test.go

```diff
@@ -234,3 +238,110 @@ func TestMultilineWhat(t *testing.T) {
 	require.Error(t, w7.UnmarshalTOML([]byte(`nope`)))
 	require.Equal(t, MultilineMatchWhichLine(-1), w7)
 }
+
+func TestMultiLineQuoted(t *testing.T) {
+	tests := []struct {
+		name      string
+		quotation string
+		quote     string
+		filename  string
+	}{
+		{
+			name:      "single-quotes",
+			quotation: "single-quotes",
+			quote:     `'`,
+			filename:  "multiline_quoted_single.csv",
+		},
+		{
+			name:      "double-quotes",
+			quotation: "double-quotes",
+			quote:     `"`,
+			filename:  "multiline_quoted_double.csv",
+		},
+		{
+			name:      "backticks",
+			quotation: "backticks",
+			quote:     "`",
+			filename:  "multiline_quoted_backticks.csv",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			expected := []string{
+				`1660819827410,1,some text without quotes,A`,
+				fmt.Sprintf(`1660819827411,1,%ssome text all quoted%s,A`, tt.quote, tt.quote),
+				fmt.Sprintf(`1660819827412,1,%ssome text all quotedbut wrapped%s,A`, tt.quote, tt.quote),
```

Comment:
Wait, the newlines are removed from the result? Is that expected behaviour?

Reply:
Well that's what tail does. Do you think this is a problem?

Reply:
In a CSV, you quote the field i it is a string, and when that string has newlines, you want those to remain in the processed metric field, right?

Reply:
I see. Will try to realize it then...